### DISCUSSION TITLE
Дочерние процессы ⚡️ Node.js с примерами кода

### DISCUSSION THREAD

**Discussion by giscus** [xsltdev/nodejsdev.ru#58] [Announcements]
# Дочерние процессы ⚡️ Node.js с примерами кода

Справочник Node.js с примерами кода

https://nodejsdev.ru/guides/webdraftt/child-processes/

**Comment by Bahurtsev** [comment]
У вас на слушателях событий нет фигурных скобок. Без них код не запустится.

Сейчас так:

child.on('message', (code) =>
    console.log(`Message to parent: ${code}`)
);

А надо так:

child.on('message', (code) => {
    console.log(`Message to parent: ${code}`);
});

**Reply by bndby** [reply]
Ну т. е. Вы попробовали запустить этот код и он вызвал ошибку, верно? А какая ошибка у вас была?

**Comment by YuraKissil** [comment]
Прошу исправить "потребялемых" -> "потребляемых"
